onnxruntime/test/testdata/ort_github_issue_17000.py

```diff
@@ -0,0 +1,77 @@
+import numpy as np
+import onnx
+from onnx import TensorProto, helper, numpy_helper
+
+
+def order_repeated_field(repeated_proto, key_name, order):
+    order = list(order)
+    repeated_proto.sort(key=lambda x: order.index(getattr(x, key_name)))
+
+
+def make_node(op_type, inputs, outputs, name=None, doc_string=None, domain=None, **kwargs):
+    node = helper.make_node(op_type, inputs, outputs, name, doc_string, domain, **kwargs)
+    if doc_string == "":
+        node.doc_string = ""
+    order_repeated_field(node.attribute, "name", kwargs.keys())
```

Comment:
just curious, why do we need this?

Reply:
I pulled some things out of the original model with onnx2py and left those parts as-is. probably not needed here though